crates/dslab-mp/src/process.rs

```diff
@@ -1,7 +1,15 @@
+use std::fmt::Debug;
+use downcast_rs::{Downcast, impl_downcast};
+use dyn_clone::{clone_trait_object, DynClone};
+
 use crate::context::Context;
 use crate::message::Message;
 
-pub trait Process {
+pub trait ProcessState: Downcast + Debug {}
+
+impl_downcast!(ProcessState);
+
```

Comment:
давай такое сделаем чтоб по умолчанию было?
```suggestion
impl ProcessState for String {

}
```
вроде у меня без такого не завелось

Reply:
Сделал дефолтную имплементацию с `ProcessStateStub`